.github/workflows/build_docker_image.yml

```diff
@@ -1,12 +1,10 @@
 name: Build Docker Image
 
 on:
+  workflow_dispatch:
   push:
-    branches:
-      - main
-  pull_request_target:
-    branches:
-      - main
+    tags:
+      - '*.*.*'
```

Comment:
There's no need to trigger CI on tags. Based on our current policy, tags are created automatically and then docker image is built immediately after that.

Given that most of the existing PRs are docs related, I agree we can switch to manual triggering.